### DISCUSSION TITLE
Custom sort parameter name for QueryBuilder

### DISCUSSION THREAD

**Discussion by ManukMinasyan** [spatie/laravel-query-builder#719] [Q&A]
Do we have the ability to change the sort parameter name for each query builder?

Example:
```php
QueryBuilder::for($currentBoard->feedbacks())
->parameter(['sort' => 'feedbackSort'])
```

Thank you.

**Comment by dominikb** [comment]
No, currently there is no way to do that as the name for sort parameters gets pulled directly from the configuration:
https://github.com/spatie/laravel-query-builder/blob/080d923aabef57a13e6c40a229bc55b6204e41a2/src/QueryBuilderRequest.php#L78

**Reply by ManukMinasyan** [reply]
I created a pull request for this.
I hope I can finish this in a few days.

[https://github.com/spatie/laravel-query-builder/pull/724](https://github.com/spatie/laravel-query-builder/pull/724)
